tracer/build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -92,6 +92,7 @@ static IEnumerable<CallTargetDefinitionSource> GetCallTargetIntegrations(ICollec
                                              TargetAssembly = assemblyNames,
                                              TargetType = GetPropertyValue<string>(attribute, "TypeName"),
                                              TargetMethod = GetPropertyValue<string>(attribute, "MethodName"),
+                                             TargetMethodArgumentsToLoad = (GetPropertyValue<ushort[]>(attribute, "TargetMethodArgumentsToLoad") ?? Enumerable.Empty<ushort>()).ToArray(),
```

Comment:
I feel like we probably want to treat null different to an empty list due to comment below, but if not, this is a bit simpler:
```suggestion
                                             TargetMethodArgumentsToLoad = GetPropertyValue<ushort[]>(attribute, "TargetMethodArgumentsToLoad") ?? Array.Empty<ushort>(),
```

Reply:
I'll remove the `??` operator so that this can return both `null` and `Array.Empty<ushort>`

Reply:
Fixed in 59aa8f2e09d6e849544b4cb7b74474075e4a930a